identity/test/services/DiscussionApiServiceTest.scala

```diff
@@ -1,118 +1,54 @@
 package services
 
-import conf.IdentityConfiguration
-import org.mockito.Matchers.any
+import clients.{DiscussionClient, DiscussionProfile, DiscussionProfileResponse, DiscussionProfileStats}
 import org.mockito.Mockito._
+import org.scalatest.AsyncFlatSpec
 import org.scalatest.Matchers._
 import org.scalatest.mockito.MockitoSugar
-import org.scalatest.{AsyncFlatSpec, EitherValues}
-import play.api.libs.ws.{WSClient, WSRequest, WSResponse}
-import scala.concurrent.Future
-import play.api.libs.json.Json
-
-class DiscussionApiServiceTest extends AsyncFlatSpec
-                               with EitherValues
-                               with MockitoSugar {
-
-  private val configMock = mock[IdentityConfiguration]
-  private val wsClientMock = mock[WSClient]
-  private val wsRequestMock = mock[WSRequest]
-  private val wsResponseMock = mock[WSResponse]
-  private val dapiService = new DiscussionApiService(wsClientMock, configMock)
-
-  val dapiApiUrl = "https://dapimock.com"
-  val testUserId = "10000001"
-
-  val valid404Response =
-    """
-      |{
-      |"status": "error",
-      |"statusCode": 404,
-      |"message": "No matching user found.",
-      |"errorCode": "USER_NOT_FOUND"
-      |}
-    """.stripMargin
-
-  val validDapiResponseWithComments =
-    """
-      |{
-      |"status": "ok",
-      |"comments": 10,
-      |"pickedComments": 1
-      |}
-    """.stripMargin
-
-  val validDapiResponseWithOutComments =
-    """
-      |{
-      |"status": "ok",
-      |"comments": 0,
-      |"pickedComments": 0
-      |}
-    """.stripMargin
 
-  val invalidDapiProfileStatsResponse =
-    """
-      |{
-      |"status": "ok",
-      |"pickedComments": 0
-      |}
-    """.stripMargin
+import scala.concurrent.Future
 
-  val validDapiResponseProfileStatsResponseWithErrorInComments =
-    """
-      |{
-      |"status": "ok",
-      |"comments": -5,
-      |"pickedComments": 0
-      |}
-    """.stripMargin
+class DiscussionApiServiceTest extends AsyncFlatSpec with MockitoSugar {
+  private def buildFixtures() = new {
```

Comment:
I think it's best to generate a new set of ScalaTest fixtures for each test to avoid reusing the mutable mockito mocks across different tests.